conf/etc/v2/cli_schema.json

```diff
@@ -0,0 +1,121 @@
+{
+    "version": "2.0.0",
+    "cluster": {
+        "help": {
+            "usage": "cortx cluster [-h | --help]",
+            "class": "ha.cli.exec.commandExecutor.ClusterCLIUsage"
+        },
+        "start": {
+            "usage": "cortx cluster start [-h] [--all | --server] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStartExecutor"
+        },
+        "stop": {
+            "usage": "cortx cluster stop [-h] [--all | --server] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStopExecutor"
+        },
+        "restart": {
+            "usage": "cortx cluster restart",
+            "class": "ha.cli.exec.clusterExecutor.ClusterRestartExecutor"
+        },
+        "standby": {
+            "usage": "cortx cluster standby [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStandbyExecutor"
+        },
+        "active": {
+            "usage": "cortx cluster active [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterActiveExecutor"
+        },
+        "list": {
+            "usage": "cortx cluster list [nodes|storagesets|services]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterListExecutor"
+        },
+        "status": {
+            "usage": "cortx cluster status [all|hw|services] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStatusExecutor"
+        },
+        "add_node": {
```

Comment:
was there any issue with "add node"?

Reply:
"cortx cluster add node" is the CLI specified in HLD. 
So it should not be changed without discussion and HLD update

Reply:
Updated command

Reply:
ok